### PR TITLE
Add easily and important aliases.

### PR DESCRIPTION
**Pull request by Bryan-0**
Add some aliases that can be useful and fast to type!

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -55,8 +55,9 @@ var commands = exports.commands = {
 			this.sendReply("Avatar changed to:\n" +
 				'|raw|<img src="//play.pokemonshowdown.com/sprites/trainers/' + avatar + '.png" alt="" width="80" height="80" />');
 		}
-	},
-
+	}, 
+	
+        lo: 'logout',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: spacing e.e

File: commands.js [hunk 2]
```diff
@@ -174,9 +175,10 @@ var commands = exports.commands = {
 		if (!user.ignorePMs) return this.sendReply("You are not blocking Private Messages!");
 		user.ignorePMs = false;
 		return this.sendReply("You are no longer blocking Private Messages.");
-	},
-
-	makechatroom: function (target, room, user) {
+	}, 
+	
+        mcr: 'makechatroom',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: eee.eee spacing

File: commands.js [hunk 3]
```diff
@@ -185,8 +187,9 @@ var commands = exports.commands = {
 			return this.sendReply("The room '" + target + "' was created.");
 		}
 		return this.sendReply("An error occurred while trying to create the room '" + target + "'.");
-	},
-
+	}, 
+	
+        dcr: 'deregisterchatroom',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: E.E

File: commands.js [hunk 4]
```diff
@@ -237,8 +240,9 @@ var commands = exports.commands = {
 				Rooms.global.writeChatRoomData();
 			}
 		}
-	},
-
+	}, 
+	
+        mj: 'modjoin',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: again, spacing....

and I don't think this is relevant

File: commands.js (hunk 2 above)
Comment (RhythmsAreFromVibrations) [inline commands.js]: necessary?

File: commands.js [hunk 5]
```diff
@@ -633,13 +637,15 @@ var commands = exports.commands = {
 			if (room.bannedUsers[altId]) delete room.bannedUsers[altId];
 		}
 		this.privateModCommand("(" + targetUser.name + "'s alts were also unbanned from room " + room.id + ": " + alts.join(", ") + ")");
-	},
-
+	}, 
+	
+        aj: 'autojoin',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: read up

File: commands.js [hunk 6]
```diff
@@ -633,13 +637,15 @@ var commands = exports.commands = {
...
 	autojoin: function (target, room, user, connection) {
 		Rooms.global.autojoinRooms(user, connection);
 	},
 
-	joim: 'join',
+	joim: 'join', 
+	jn: 'join',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: read up

File: commands.js [hunk 7]
```diff
@@ -843,8 +849,9 @@ var commands = exports.commands = {
 		this.add('|unlink|hide|' + this.getLastIdOf(targetUser));
 
 		targetUser.lock();
-	},
-
+	}, 
+	
+        ul: 'unlock',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: too similar to url tbh

**Comment by Bryan-0** [comment]
I know that, but i already try it and when im on edit there appears that there is not spacing. @RhythmsAreFromVibrations

File: commands.js [hunk 8]
```diff
@@ -896,8 +903,9 @@ var commands = exports.commands = {
 
 		this.add('|unlink|hide|' + this.getLastIdOf(targetUser));
 		targetUser.ban();
-	},
-
+	}, 
+	
+        ub: 'unban',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: this doesn't seem necessary

File: commands.js [hunk 9]
```diff
@@ -910,8 +918,9 @@ var commands = exports.commands = {
 		} else {
 			this.sendReply("User '" + target + "' is not banned.");
 		}
-	},
-
+	}, 
+	
+        uba: 'unbanall',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: uba looks like Ubahs, which then looks like Ubers

read up

File: commands.js [hunk 10]
```diff
@@ -923,8 +932,9 @@ var commands = exports.commands = {
 			delete Users.lockedIps[i];
 		}
 		this.addModCommand("All bans and locks have been lifted by " + user.name + ".");
-	},
-
+	}, 
+	
+        bi: 'banip',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``biweekly?``

read up

File: commands.js [hunk 11]
```diff
@@ -936,8 +946,9 @@ var commands = exports.commands = {
 
 		Users.bannedIps[target] = '#ipban';
 		this.addModCommand("" + user.name + " temporarily banned the " + (target.charAt(target.length - 1) === '*' ? "IP range" : "IP") + ": " + target);
-	},
-
+	}, 
+	
+        ubi: 'unbanip',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: read up

File: commands.js [hunk 12]
```diff
@@ -950,8 +961,9 @@ var commands = exports.commands = {
 		}
 		delete Users.bannedIps[target];
 		this.addModCommand("" + user.name + " unbanned the " + (target.charAt(target.length - 1) === '*' ? "IP range" : "IP") + ": " + target);
-	},
-
+	}, 
+	
+        rl: 'rangelock',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: i``rl``

read up

File: commands.js [hunk 13]
```diff
@@ -963,8 +975,9 @@ var commands = exports.commands = {
 
 		Users.lockRange(range, isIp);
 		this.addModCommand("" + user.name + " temporarily locked the range " + range + ".");
-	},
-
+	}, 
+	
+        rul: 'rangeunlock',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``rul``es

read up

File: commands.js [hunk 14]
```diff
@@ -1035,8 +1048,9 @@ var commands = exports.commands = {
 		}
 
 		if (targetUser) targetUser.updateIdentity();
-	},
-
+	}, 
+	
+        fp: 'forcepromote',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``fps``

read up

File: commands.js [hunk 15]
```diff
@@ -1059,8 +1073,9 @@ var commands = exports.commands = {
 	deroomauth: 'roomdeauth',
 	roomdeauth: function (target, room, user) {
 		return this.parse('/roomdemote ' + target + ', deauth');
-	},
-
+	}, 
+	
+        mc: 'modchat',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``m``ine``c``raft?

read up

File: commands.js [hunk 16]
```diff
@@ -1191,8 +1206,9 @@ var commands = exports.commands = {
 		Rooms.global.cancelSearch(targetUser);
 		targetUser.resetName();
 		targetUser.send("|nametaken||" + user.name + " considers your name inappropriate" + (reason ? ": " + reason : "."));
-	},
-
+	}, 
+	
+        ml: 'modlog',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``mlk shke``

Milk Shakes y/n

read up

File: commands.js [hunk 17]
```diff
@@ -1280,6 +1296,7 @@ var commands = exports.commands = {
 	 * Server management commands
 	 *********************************************************/
 
+        hp: 'hotpatch',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: Mismagius has a base 60 ``hp``.

read up

File: commands.js [hunk 18]
```diff
@@ -1351,8 +1368,9 @@ var commands = exports.commands = {
 		if (!this.can('hotpatch')) return false;
 		fs.writeFile('data/learnsets.js', 'exports.BattleLearnsets = ' + JSON.stringify(Tools.data.Learnsets) + ";\n");
 		this.sendReply("learnsets.js saved.");
-	},
-
+	}, 
+	
+        dl: 'disableladder',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``d``own``l``oad

you should get it by now

File: commands.js [hunk 19]
```diff
@@ -1361,8 +1379,9 @@ var commands = exports.commands = {
 		LoginServer.disabled = true;
 		this.logModCommand("The ladder was disabled by " + user.name + ".");
 		this.add("|raw|<div class=\"broadcast-red\"><b>Due to high server load, the ladder has been temporarily disabled</b><br />Rated games will no longer update the ladder. It will be back momentarily.</div>");
-	},
-
+	}, 
+	
+        el: 'enableladder'
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``el``etist

File: commands.js [hunk 20]
```diff
@@ -1512,14 +1531,16 @@ var commands = exports.commands = {
 			Users.checkRangeBanned = Cidr.checker(rangebans);
 			connection.sendTo(room, "ipbans.txt has been reloaded.");
 		});
-	},
-
+	}, 
+	
+        rp: 'refreshpage',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: ``rp`` is the initials for Role Play

File: commands.js [hunk 21]
```diff
@@ -1512,14 +1531,16 @@ var commands = exports.commands = {
...
 	refreshpage: function (target, room, user) {
 		if (!this.can('hotpatch')) return false;
 		Rooms.global.send('|refresh|');
 		this.logEntry(user.name + " used /refreshpage");
-	},
-
+	}, 
+	
+        us: 'updateserver',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: USA! USA!

File: commands.js [hunk 22]
```diff
@@ -1569,8 +1590,9 @@ var commands = exports.commands = {
 				CommandParser.updateServerLock = false;
 			});
 		});
-	},
-
+	}, 
+	
+        cf: 'crashfixed',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: spacing

File: commands.js [hunk 23]
```diff
@@ -1763,8 +1785,9 @@ var commands = exports.commands = {
 		if (!room.decision) return this.sendReply("You can only do this in battle rooms.");
 
 		room.decision(user, 'choose', 'team ' + target);
-	},
-
+	}, 
+	
+        jb: 'joinbattle',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: spacing

doesn't seem necessary

File: commands.js [hunk 24]
```diff
@@ -1777,8 +1800,9 @@ var commands = exports.commands = {
 		if (!room.leaveBattle) return this.sendReply("You can only do this in battle rooms.");
 
 		room.leaveBattle(user);
-	},
-
+	}, 
+	
+        kb: 'kickbattle',
```
Comment (RhythmsAreFromVibrations) [inline commands.js]: kb also seems a lot like "kickban"

**Comment by RhythmsAreFromVibrations** [comment]
And did you test the code first?

**Comment by Bryan-0** [comment]
@RhythmsAreFromVibrations  Yes i test all that.

**Comment by RhythmsAreFromVibrations** [comment]
...Did it work?

I'd suggest testing the code and then fixing it until it works before trying to make a commit/PR.

**Comment by Bryan-0** [comment]
@RhythmsAreFromVibrations  I test it on localhost, and yes that works correctly.

**Comment by Bryan-0** [comment]
And i think that aliases is good, because will be easily to type and do not type all the command.

**Comment by RhythmsAreFromVibrations** [comment]
Even if it's easier to type, does it make sense? like Hp is also the initials for Hit Points for Pokemon

lo is the initials for Life Orb, a Pokemon Item

it'd commonly confuse people what they mean, and some of them aren't dignified enough

**Comment by RhythmsAreFromVibrations** [comment]
And if you've tested everything correctly, you wouldn't have three failed CI builds.

**Comment by ascriptmaster** [comment]
Big mod commands like `/hotpatch` don't need aliases because we don't want admins to quick-type them anyways. If you're using a command as heavy as hotpatch, you really should type the whole thing out so that you don't accidentally hotpatch and lag the server, especially since "HP" also can stand for Hit Points or Hidden Power! "Oh, whoops, accidentally created a chat room!" I especially don't want to accidentally type `/rangeunlock` if I press enter too soon when using the command `/rules`, or something like that. 

Half of these commands are far too short for mod-level commands. If you're using such commands, you have to be serious about it (rangebanning is a HUGE thing). Not only that, some of the abbreviations usually mean other things (dl = download, rp = roleplaying, etc.)

Also, you removed our `/fr` (`/forcerename`) alias, which is a command mods actually need to type quickly to stop trolls.

I understand that you want to help, but adding random aliases to commands like this is not going to be beneficial to the server.

**Comment by Zarel** [comment]
Zero of these appear to be a good idea.

**Pull request closed without merging**